src/20231116_Multi_IncreaseStablecoinOptimalBorrowRates/IncreaseStablecoinOptimalBorrowRates.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Increase Stablecoin Optimal Borrow Rates"
+author: "Gauntlet, Block Analitica"
+discussions: "https://governance.aave.com/t/arfc-increase-optimal-borrow-rates-for-ethereum-stablecoin-markets/15096"
+---
+
+## Summary
+
+This AIP proposes to make adjustments to stablecoin interest rate models across Aave v2 and v3 markets. Specifically, this AIP proposes to increase the slope 1 parameter for stablecoins to implement a 5% optimal borrow rate.
+
+## Specification
+
+| Market       | Asset  | Parameter          | Current Value | Recommended Value |
+| ------------ | ------ | ------------------ | ------------- | ----------------- |
+| Ethereum v2  | USDC   | variableRateSlope1 | 0.035         | 0.05              |
```

Comment:
so this actually needs to be changed then for the hackmd. all v2 USDC current values = 0.04